modules/Score.js

```diff
@@ -1,6 +1,6 @@
 export default class Score {
-  constructor(name, score) {
```

Comment:
- [ ] Great job so far 👏🏻 However I was not able to run perfectly your app on my local computer instead I got an empty page 👇🏻 Could you please fix this or add more information to your README file on how to run your app locally 👍🏻 

![Screenshot from 2023-02-02 03-34-31](https://user-images.githubusercontent.com/79058364/216209997-1f68acf6-eebc-4739-8792-372b58f1cc27.png)